packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -590,24 +580,6 @@ export async function setTransformationToRunningState() {
         convertDateToTimestamp(new Date(CodeTransformTelemetryState.instance.getStartTime()))
     )
 
-    const projectPath = transformByQState.getProjectPath()
-    let projectId = telemetryUndefined
-    if (projectPath !== undefined) {
-        projectId = getStringHash(projectPath)
-    }
-
-    telemetry.codeTransform_jobStartedCompleteFromPopupDialog.emit({
```

Comment:
Question: I assumed this is deadcode since we have all form fields in the chat now correct.